tests/integration/metrics/api/decorators/test_auth.py

```diff
@@ -0,0 +1,115 @@
+from http import HTTPStatus
+from unittest import mock
+
+import pytest
+from django.urls import path
+from rest_framework.test import APIClient
+from rest_framework.views import APIView
+from django.test import override_settings
+from metrics.api.decorators.auth import require_authorisation, RBAC_AUTH_X_HEADER
+from django.http import JsonResponse
+
+from tests.factories.metrics.rbac_models.rbac_group_permissions import (
+    RBACPermissionGroupFactory,
+)
+from tests.factories.metrics.rbac_models.rbac_permission import RBACPermissionFactory
+
+
+MODULE_PATH = "metrics.api.decorators.auth"
+
+
+class MockApiView(APIView):
```

Comment:
```suggestion
class FakeApiView(APIView):
```
Nitpick: I know terminology varies a lot but we tend to distinguish between flexible mock objects like the ones you'd get from the `unittest` library and fakes. In this case, we'd say this is a fake because its a barebone implementation of the real thing.